bin/blackmagic.py

```diff
@@ -609,7 +627,10 @@ def resolve(self, request, packages_list):
                   '/var/lib/dpkg/status'
         ] + packages_list
 
-        data = yield util.execute_async(command_line)
+        command_data = ' '.join(command_line)
+        # data = yield util.execute_async(command_line)
+        data = subprocess.Popen(command_data.split(), stdout=subprocess.PIPE)
+        output, error = data.communicate()
```

Comment:
@leocoan don't forget about error handling. If an error has occurred, this code should notify the client